services/mac_manager/src/main/java/com/futurewei/alcor/macmanager/service/implement/MacServiceImpl.java

```diff
@@ -303,6 +300,20 @@ private boolean isValidRange(MacRange macRange) {
         return from < to;
     }
 
+    public MacRange createDefaultRange(String oui) {
```

Comment:
If createDefaultRange, updateMacAllocationStatus and macToIndex are only used by methods inside MacServiceImpl, then let us mark them private.

Reply:
They are changed.